rpc/cloudagent/compute/virtualmachine/moc_cloudagent_virtualmachine.proto

```diff
@@ -103,6 +103,12 @@ message SecurityConfiguration {
 	SecurityType securityType = 3;
 }
 
+message AvailabilitySetReference {
+	string groupName = 1;
+	string name = 2;
+	string realizedName = 3;
```

Comment:
Nit: Why are we using realized name instead of name?

Reply:
See previous thread with Pradeep